scene/main/node.cpp

```diff
@@ -1868,18 +1868,31 @@ bool Node::has_persistent_groups() const {
 
 	return false;
 }
-void Node::_print_tree(const Node *p_node) {
+void Node::_print_tree_pretty(const String prefix, const bool last) {
```

Comment:
Are those arguments really useful if they're not exposed? Best keep it simple if they're not meant to be used.

Reply:
It's a recursive function, so while they're not exposed, they are needed: eg for indentation.

Reply:
In that case, shouldn't you have a private helper function that does all the recursion?

Reply:
Hence the difference between print_tree_pretty and _print_tree_pretty. Or am I missing something?